Java/main-sample/pom.xml

```diff
@@ -99,7 +99,7 @@
         <dependency>
             <groupId>commons-io</groupId>
             <artifactId>commons-io</artifactId>
-            <version>2.4</version>
+            <version>RELEASE</version>
```

Comment:
I wouldn't do it. We will spend time every new release fixing the code and we won't understand why something broke.
You need to control the dependency versions. 
Otherwise why we don't do it in our own code? 
You want to plan updating versions and testing it, not be surprised every time you run build to wonder why the hell it suddenly fails without any code change

Reply:
ok sure